blocks/init/src/Blocks/components/image/image-style.scss

```diff
@@ -15,7 +15,84 @@
 .image-wrap {
 	display: flex;
 
-	&__align {
-		@include flex-horizontal-align-modifiers;
+	&[data-align='top left'] {
```

Comment:
This would be a good candidate for some kind of placeholder if we will use the matrix alignment in other components some day.